stl/inc/xstring

```diff
@@ -3883,6 +3881,7 @@ public:
 #endif // _HAS_CXX20
         {
             if (_Growth <= _Mypair._Myval2._Myres - _Old_size) { // growth fits
+                _ASAN_STRING_MODIFY(*this, _Old_size, _Old_size + _Growth);
                 _Mypair._Myval2._Mysize = _Old_size + _Growth;
                 _Elem* const _Old_ptr   = _Mypair._Myval2._Myptr();
                 _Elem* const _Insert_at = _Old_ptr + _Off;

```

Comment:
`replace(const size_type _Off, size_type _Nx, const size_type _Count, const _Elem _Ch)` is missing annotation coverage